assets/src/edit-story/components/library/test/text/textPane.js

```diff
@@ -1,92 +0,0 @@
-/*
- * Copyright 2020 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-/**
- * External dependencies
- */
-import { act, fireEvent } from '@testing-library/react';
-import { FlagsProvider } from 'flagged';
-
-/**
- * Internal dependencies
- */
-import TextPane from '../../panes/text/textPane';
-import { DEFAULT_PRESET } from '../../panes/text/textPresets';
-import { renderWithTheme } from '../../../../testUtils/index';
-
-jest.mock('../../useLibrary');
-jest.mock('../../../../app/font/useFont');
-import useFont from '../../../../app/font/useFont';
-import useLibrary from '../../useLibrary';
-import FontContext from '../../../../app/font/context';
-import fontsListResponse from '../../../fontPicker/test/fontsResponse.json';
-import { curatedFontNames } from '../../../../app/font/curatedFonts';
-
-describe('TextPane', () => {
```

Comment:
Wait, why are these tests being removed? Could they not simply be amended?

Reply:
I removed it cus it's just the one test for the `Add new text` button that was removed.

But yea, looking at it again I can just perform the same test but on clicking a preset instead of the `Add new text button. will do that unless you think otherwise